src/pybind/mgr/dashboard_v2/module.py

```diff
@@ -74,6 +79,20 @@ def __init__(self, *args, **kwargs):
         mgr.init(self)
         self._url_prefix = ''
 
+    def can_run(self):
+        if cherrypy is None:
```

Comment:
we also need "bcrypt" package for the authentication to work, can we also add that check to this method?